llamabot.py

```diff
@@ -100,4 +100,20 @@
                 except IndexError:
                     pass
 
-
+    # ?tell
+    with open("msg_store", 'r') as msg_store:
+        l_msg_store = msg_store.readlines()
+    del_msg, msg_list = [], []
+    for line in range(0, len(l_msg_store), 4):
+        if "{}\n".format(sender.lower()) == l_msg_store[line].lower():
+            del_msg.append(line)
+            msg_list.append("{} told me to tell you this, {}: {}".
+                format(l_msg_store[n + 2].rstrip(), sender, l_msg_store[n + 1].rstrip()))
+    for line in range(len(del_msg)):
+        l_msg_store[del_msg[line]-4*line : del_msg[line]+4*(1-line)] = []
+    with open("msg_store", 'w') as msg_store:
+        msg_store.write(''.join(l_msg_store))
+    if not target.lstrip('@+').startswith('#'):
```

Comment:
target is not defined yet

Reply:
Defined target out of loop.

Reply:
Putting the code I added in llamabot.py in actions.py will be a problem. If someone sends a command, then pending messages won't be sent if there are any for that user. There may also be more than 1 pending message. So, leave code as is (could put the ?tell part in a function)?